tests/integration/fixtures.py

```diff
@@ -0,0 +1,131 @@
+import logging
+
+import pytest
+
+from localstack.utils.aws import aws_stack
+from localstack.utils.aws.aws_stack import create_dynamodb_table
+from localstack.utils.common import short_uid
+
+try:
+    import botostubs
```

Comment:
nit: Perhaps we can add a line to `requirements.txt` to install `botostubs` automatically into dev environments?
```
botostubs                  #extended-lib
```